indi-sv305/sv305_ccd.cpp

```diff
@@ -269,6 +237,15 @@ bool Sv305CCD::Connect()
     // wait a bit for the camera to get ready
     usleep(0.5 * 1e6);
 
+    // disable suto save param
+    status = SVBSetAutoSaveParam(cameraID, SVB_FALSE);
+    if (status != SVB_SUCCESS)
+    {
+        LOG_ERROR("Error, disable auto save param failed.\n");
```

Comment:
No need for \n

Reply:
I will remove '\n'.

I have a concern.
Since before I contributed, the LOG_ERROR in 'sv305_ccd.cpp' contains '\n' in all but one place.

Should I remove '\n' from these as well?
*Not at this time.

Reply:
Right when using LOG_XXX do not include new line.